src/Game.h

```diff
@@ -184,6 +191,498 @@ class Game
+	void VmStartImmersion();                              // 429 START_IMMERSION
+	void VmStopImmersion();                               // 430 STOP_IMMERSION
+	void VmStopAllImmersion();                            // 431 STOP_ALL_IMMERSION
+	void VmSetCreatureInTemple();                         // 432 SET_CREATURE_IN_TEMPLE
+	void VmGameDrawText();                                // 433 GAME_DRAW_TEXT
+	void VmGameDrawTempText();                            // 434 GAME_DRAW_TEMP_TEXT
+	void VmFadeAllDrawText();                             // 435 FADE_ALL_DRAW_TEXT
+	void VmSetDrawTextColour();                           // 436 SET_DRAW_TEXT_COLOUR
+	void VmSetClippingWindow();                           // 437 SET_CLIPPING_WINDOW
+	void VmClearClippingWindow();                         // 438 CLEAR_CLIPPING_WINDOW
+	void VmSaveGameInSlot();                              // 439 SAVE_GAME_IN_SLOT
+	void VmSetObjectCarrying();                           // 440 SET_OBJECT_CARRYING
+	void VmPosValidForCreature();                         // 441 POS_VALID_FOR_CREATURE
+	void VmGetTimeSinceObjectAttacked();                  // 442 GET_TIME_SINCE_OBJECT_ATTACKED
+	void VmGetTownAndVillagerHealthTotal();               // 443 GET_TOWN_AND_VILLAGER_HEALTH_TOTAL
+	void VmGameAddForBuilding();                          // 444 GAME_ADD_FOR_BUILDING
+	void VmEnableDisableAlignmentMusic();                 // 445 ENABLE_DISABLE_ALIGNMENT_MUSIC
+	void VmGetDeadLiving();                               // 446 GET_DEAD_LIVING
+	void VmAttachSoundTag();                              // 447 ATTACH_SOUND_TAG
+	void VmDetachSoundTag();                              // 448 DETACH_SOUND_TAG
+	void VmGetSacrificeTotal();                           // 449 GET_SACRIFICE_TOTAL
+	void VmGameSoundPlaying();                            // 450 GAME_SOUND_PLAYING
+	void VmGetTemplePosition();                           // 451 GET_TEMPLE_POSITION
+	void VmCreatureAutoscale();                           // 452 CREATURE_AUTOSCALE
+	void VmGetSpellIconInTemple();                        // 453 GET_SPELL_ICON_IN_TEMPLE
+	void VmGameClearComputerPlayerActions();              // 454 GAME_CLEAR_COMPUTER_PLAYER_ACTIONS
+	void VmGetFirstInContainer();                         // 455 GET_FIRST_IN_CONTAINER
+	void VmGetNextInContainer();                          // 456 GET_NEXT_IN_CONTAINER
+	void VmGetTempleEntrancePosition();                   // 457 GET_TEMPLE_ENTRANCE_POSITION
+	void VmSaySoundEffectPlaying();                       // 458 SAY_SOUND_EFFECT_PLAYING
+	void VmSetHandDemoKeys();                             // 459 SET_HAND_DEMO_KEYS
+	void VmCanSkipTutorial();                             // 460 CAN_SKIP_TUTORIAL
+	void VmCanSkipCreatureTraining();                     // 461 CAN_SKIP_CREATURE_TRAINING
+	void VmIsKeepingOldCreature();                        // 462 IS_KEEPING_OLD_CREATURE
+	void VmCurrentProfileHasCreature();                   // 463 CURRENT_PROFILE_HAS_CREATURE
```

Comment:
This addition is a no-go for us.
The game class is already too monolithic and we're trying to get rid of the game class entirely.
This addition add too much coupling and complexity to the game class.
You may want to look at how we handle Scripts.

A better solution would be to move this to a [locator service pattern](https://gameprogrammingpatterns.com/service-locator.html). See Locator.h.